node/test/lib/alloc-cluster.js

```diff
@@ -167,6 +173,31 @@ allocCluster.test.only = function testClusterOnly(desc, opts, t) {
     });
 };
 
+function connectChannels(channels, callback) {
+    return parallel(channels.map(function (channel) {
```

Comment:
works for now, realize I'll probably pivot it later to just use the existing async abstraction, the counted ready signal